packages/xwayland.rb

```diff
@@ -39,5 +40,6 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "ln", "-s", "#{CREW_PREFIX}/bin/Xwayland", "#{CREW_DEST_PREFIX}/bin/X"
```

Comment:
This looks good.  Can we bump the version?
```
- version '1.19.3'
+ version '1.19.3-1'
```

Reply:
Done.